tests/k6/tests/serviceowner/concurrency.js

```
@@ -16,14 +16,14 @@ export default function () {
 
     describe('Attempt to add a few child entities concurrently', async () => {
 
-        let promises = [];
+        let batch = [];
 
         for (let i=0; i<10; i++) {
             let activity = { type: "Information", performedBy: { actorType: "serviceOwner" }, description: [ { value: i.toString(), languageCode: "nb"}]};
-            promises.push(postSOAsync('dialogs/' + dialogId + '/activities?' + i, activity))
+            batch.push(['dialogs/' + dialogId + '/activities?' + i, activity]);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)